scripts/known_good.json

```diff
@@ -53,13 +53,16 @@
             "sub_dir": "jsoncpp",
             "build_dir": "jsoncpp/build",
             "install_dir": "jsoncpp/build/install",
-            "commit": "1.9.5",
+            "commit": "1.9.6",
             "cmake_options": [
                 "-DCMAKE_POSITION_INDEPENDENT_CODE=ON",
                 "-DJSONCPP_WITH_TESTS=OFF",
                 "-DJSONCPP_WITH_POST_BUILD_UNITTEST=OFF",
                 "-DJSONCPP_WITH_WARNING_AS_ERROR=OFF",
-                "-DJSONCPP_WITH_PKGCONFIG_SUPPORT=OFF"
+                "-DJSONCPP_WITH_PKGCONFIG_SUPPORT=OFF",
+                "-DBUILD_SHARED_LIBS=OFF",
+                "-DBUILD_STATIC_LIBS=ON",
+                "-DBUILD_OBJECT_LIBS=OFF"
```

Comment:
Lines up with LunarG/VulkanTools and helps avoid building unnecessary libs. Profiles only cares about the static lib.